src/css/components.css

```diff
@@ -0,0 +1,673 @@
+      font-size: 5.96em;
+    }
+  }
+
+  .type-9 {
+    font-size: 3.52em;
+    letter-spacing: -0.026em;
+    @media (min-width: 768px) {
+      font-size: 5.18em;
+    }
+    @media (min-width: 992px) {
+      font-size: 7.45em;
+    }
+  }
+
+  .types
+  .splash-text {
+    line-height: var(--leading-display);
+    font-weight: var(--font-weight-bold);
+    margin-bottom: 0.6em;
+    clear: both;
+
+    a {
+      text-decoration: none;
+      font-weight: var(--font-weight-bold);
+    }
+  }
+
+  .splash-text {
+    font-size: 2.31em;
+    letter-spacing: -0.02em;
+
+    @media (min-width: 768px) {
+      font-size: 2.99em;
+    }
+
+    @media (min-width: 992px) {
+      font-size: 3.81em;
+    }
+
+    a {
+      text-decoration: none;
+      font-weight: var(--font-weight-bold);
+    }
+  }
+
+  .intro-text {
+    font-size: 1.32em;
+    letter-spacing: -0.012em;
+    font-weight: var(--leading-normal);
+    max-width: var(--max-width-prose);
+
+    @media (min-width: 768px) {
+      font-size: 1.44em;
+    }
+
+    @media (min-width: 992px) {
+      font-size: 1.56em;
+    }
+  }
+
+  .quote-text {
+    font-size: 1.25em;
+    font-style: italic;
+  }
+
+  .caption,
+  .credits {
+    font-size: max(1.6rem, 0.9em);
+    font-weight: var(--leading-snug);
+    color: var(--color-cool-grey);
+  }
+
+  .caption {
+    margin-top: 0.4em;
+  }
+
+  .credits {
+    font-style: italic;
+  }
+
+  .subheading {
+    font-size: 1.15em;
+    letter-spacing: -0.01em;
+
+    @media (min-width: 768px) {
+      font-size: 1.2em;
+    }
+
+    @media (min-width: 992px) {
+      font-size: 1.25em;
+    }
+  }
+
+  .big-paragraph {
+    font-size: 1.15em;
+    font-weight: var(--leading-cozy);
+
+    @media (min-width: 768px) {
+      font-weight: var(--leading-normal);
+    }
+  }
+
+  .card-paragraph {
+    font-size: max(1.6rem, 0.93em);
+    font-weight: var(--leading-snug);
+
+    @media (min-width: 768px) {
+      font-weight: var(--leading-cozy);
+    }
+  }
+
+  .basefont-19 {
+    font-size: 1.6rem;
+
+    @media (min-width: 768px) {
+      font-size: 1.8rem;
+    }
+
+    @media (min-width: 1500px) {
+      font-size: 1.9rem;
+    }
+  }
+
+  .basefont-20 {
+    font-size: 1.8rem;
+
+    @media (min-width: 768px) {
+      font-size: 1.9rem;
+    }
+
+    @media (min-width: 1500px) {
+      font-size: 2rem;
+    }
+  }
+
+  .basefont-21 {
+    font-size: 1.8rem;
+
+    @media (min-width: 768px) {
+      font-size: 1.9rem;
+    }
+
+    @media (min-width: 1500px) {
+      font-size: 2.1rem;
+    }
+  }
+
+  .basefont-22 {
+    font-size: 1.8rem;
+
+    @media (min-width: 768px) {
+      font-size: 2rem;
+    }
+
+    @media (min-width: 1500px) {
+      font-size: 2.2rem;
+    }
+  }
+
+  .basefont-23 {
+    font-size: 1.8rem;
+
+    @media (min-width: 768px) {
+      font-size: 2.1rem;
+    }
+
+    @media (min-width: 1500px) {
+      font-size: 2.3rem;
+    }
+  }
+
+  .wysiwyg {
+    > *:first-child {
+      margin-top: -0.25em;
+    }
```

Comment:
I guess one nice thing about moving to a CSS based config is that you can write natural css again.